nx.json

```diff
@@ -0,0 +1,53 @@
+{
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "nx/tasks-runners/default",
+      "options": {
+        "cacheableOperations": [
+          "build",
+          "lint",
+          "test",
+          "build:extension",
+          "build:storybook",
+          "build:storybook:bundle"
+        ]
+      }
+    }
+  },
+  "namedInputs": {
+    "default": ["{projectRoot}/**/*"],
+    "prod": [
+      "!{projectRoot}/**/*.spec.tsx",
+      "!{projectRoot}/**/*.test.tsx",
+      "!{projectRoot}/**/*.spec.ts",
+      "!{projectRoot}/**/*.test.ts"
+    ]
+  },
+  "targetDefaults": {
+    "build": {
+      "dependsOn": [
+        "^build"
+      ],
+      "outputs": [
+        "{projectRoot}/dist"
+      ]
+    },
+    "test": {
+      "inputs": ["default", "^prod", "{workspaceRoot}/jest.config.ts"],
```

Comment:
Shouldn't that be `{projectRoot}/jest.config.ts`

Reply:
Updated in https://github.com/SAP/open-ux-tools/pull/799/commits/974edf6288eb5b96dae4b151e2670972f298f959

Reply:
You may want to consider leaving `{workspaceRoot}/jest.config.ts`. If configuration in this file changes, those changes will propagate to project-specific `jest.config.ts` files, but only at runtime. That means that changing the root configuration will not invalidate the cache of individual projects.